packages/liquidator/index.js

```diff
@@ -86,21 +122,17 @@ async function run({
       minSponsorTokens,
       collateralTokenAddress,
       syntheticTokenAddress,
-      isExpired,
```

Comment:
I removed this call to simply the logic a bit. this info is only _really_ needed within the main while loop and it is queried there anyways.